apps/client/src/widgets/onboarding/components/step/user-info/user-info.tsx

```diff
@@ -13,28 +16,20 @@ const USER_TITLE = '기본 정보';
 const USER_DESCRIPTION = '기본 정보를 입력해주세요';
 
 interface UserInfoProps {
-  value: UserInfoStateProps;
-  onChange: (value: UserInfoStateProps) => void;
+  control: Control<z.infer<typeof onboardingFormSchema>>;
```

Comment:
```suggestion
  control: Control<onboardingFormType>;
```

```ts
export type onboardingFormType = z.infer<typeof onboardingFormSchema>;
```

이 타입을 스키마 파일에서 export 하고 있어서 요렇게 써줘도 될 것 같아요

Reply:
어머 onboardingFormType 만들어놓고 안썻네요... 반영하겠습니다 !